Quotes.js

```diff
@@ -1,68 +1,38 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
 import { useIsShake } from './useIsShake';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY } from './constants';
-
-const quotes = [
-  "onus is on us.",
-  "lawbreakers can't be legislated.",
-  "an earthly loss is an ancestral gain.",
-  "a mistake is the truth trying to come out.",
-  "overturning the bullshit on a personal level by not subscribing to it is the new movement.",
-  "there are mistakes that are derived from a lack of discipline and those that are born of brilliance.",
-  "don't go in and out of meditative spaces - stay there with that awareness, always.",
-  "everything in life is meditative.",
-  "tis better to hit an air ball, than to force a note that don't wanna come out.",
-  "the blues ain't a form. it's a feel.",
-  "art serves as a bridge between this world and the ancestral one.",
-  "art promotes creative and critical thinking, both of which are threats to the status-quo.",
-  "it's illogical to expect your abuser to aid in your recovery.",
-  "colonialism has people convinced that we must take from someone else to get ahead.",
-  "indigenous people understand that to serve the community helps everyone.",
-  "if we care about the place in which we live, we must value the artists who help make it beautiful.",
-  "we are here to assist in each other's evolution.",
-  "art is not here to make you feel good. it's here to make you grow, which is good for you.",
-  "take care of the music and the music will take care of you.",
-  "protect the people you love. call out bullshit at all costs. appreciate free thought in all its glory.",
-  "we only want to believe the 'good' things about the people we admire, when what makes them heroes is how they negotiate their flaws.",
-];
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY, quotes } from './constants';
+import { getRandomInt } from './utils';
```

Comment:
import `function getRandomInt` from `utils.js`

Reply:
don't need this comment. The code is clear on what is happening